guides/common/modules/proc_adding-lifecycle-environments-by-using-cli.adoc

```diff
@@ -1,25 +1,17 @@
 :_mod-docs-content-type: PROCEDURE
 
-[id="Adding_Lifecycle_Environments_{context}"]
-= Adding lifecycle environments to {SmartProxyServers}
+[id="adding-lifecycle-environments-by-using-cli"]
+= Adding lifecycle environments to {SmartProxyServers} by using Hammer CLI
 
+[role="_abstract"]
 If your {SmartProxyServer} has the content functionality enabled, you must add an environment so that {SmartProxy} can synchronize content from {ProjectServer} and provide content to host systems.
```

Comment:
out of scope: I am unsure if downstream users can setup a Smart Proxy Server without content feature enabled. I imagine that it's technically possible to run Smart Proxy servers without content together with Foreman+Katello, e.g. if you only provision hosts in a specific subnet, not I do not know if ATIX or Red Hat support this.